packages/pelagos/src/formFields/TextAreaField.js

```diff
@@ -10,13 +10,32 @@ import FieldHelper from './FieldHelper';
 import './TextAreaField.less';
 
 /** A text area field. */
-const TextAreaField = ({id, className, label, value, required, resize, helperText, error, onChange, ...props}) => {
+const TextAreaField = ({
+	id,
+	className,
+	label,
+	value,
+	required,
+	resize,
+	maxLength,
+	showCounter,
```

Comment:
Are you opposed to removing this property? I feel like it opens the door for inconsistency across usages of `TextAreaField`.

If the counter only shows up some of the time when `maxLength` is set, users wouldn't know if there is a max length on fields that don't have the counter (until they attempted to submit a value that is too long).